libsignal-service-actix/src/push_service.rs

```diff
@@ -120,6 +106,20 @@ impl PushService for AwcPushService {
     type ByteStream = Box<dyn futures::io::AsyncRead + Unpin>;
     type WebSocket = AwcWebSocket;
 
+    fn new(
+        cfg: impl Into<ServiceConfiguration>,
+        credentials: Option<ServiceCredentials>,
+        user_agent: &'static str,
```

Comment:
Can we make this less stringent than `&'static str`? Whisperfish will almost certainly provide a way to dynamically set a UA.